### DISCUSSION TITLE
convert: FontSubstitutionRequired `Courier' @ warning/type.c/GetTypeInfoByFamily/493.

### DISCUSSION THREAD

**Discussion by CxcPDF** [ImageMagick/ImageMagick#2850] [Help]
convert wmf to png, but there is the error:
`convert: FontSubstitutionRequired `Courier' @ warning/type.c/GetTypeInfoByFamily/493.`

I have installed libwmf and type `identify -list font` for a list of available fonts that ImageMagick discovered. My system includes Courier fonts:

```
Font: Courier
    family: Courier
    style: Normal
    stretch: Normal
    weight: 400
    glyphs: /usr/share/fonts/type1/gsfonts/n022003l.pfb
  Font: Courier-Bold
    family: Courier
    style: Normal
    stretch: Normal
    weight: 700
    glyphs: /usr/share/fonts/type1/gsfonts/n022004l.pfb
  Font: Courier-BoldOblique
    family: Courier
    style: Oblique
    stretch: Normal
    weight: 700
    glyphs: /usr/share/fonts/type1/gsfonts/n022024l.pfb
  Font: Courier-Oblique
    family: Courier
    style: Oblique
    stretch: Normal
    weight: 400
    glyphs: /usr/share/fonts/type1/gsfonts/n022023l.pfb
```

**Comment by urban-warrior** [comment]
Its possible your WMF requires the Courier font at a certain weight or style that is not available, thus the font substitution.

**Reply by CxcPDF** [reply]
How could I get the weight or style of WMF?

**Comment by lalitap25** [comment]
I am also having the same issue. Getting error message like convert: FontSubstitutionRequired `Courier' @ warning/type.c/GetTypeInfoByFamily/490 what is the solution for it?
